Week-2/Assignment2.1/main.css

```diff
@@ -0,0 +1,135 @@
+
+* {
+    margin: 0;
+    padding: 0;
+}
+body {
+    background-color: #007eae;
+    min-height: 100vh;
+    display: flex;
```

Comment:
Instead of giving the body as display: flex, It would be better if you would have created a container element.
Follow the BEM convention for naming CSS classes.